modules/nodes-cluster-enabling-features-about.adoc

```diff
@@ -24,7 +24,7 @@ The following Technology Preview features are enabled by this feature set:
 ** Shared Resources CSI Driver in OpenShift Builds. Enables the Container Storage Interface (CSI). (`CSIDriverSharedResource`)
 ** Swap memory on nodes. Enables swap memory use for {product-title} workloads on a per-node basis. (`NodeSwap`)
 ** OpenStack Machine API Provider. This gate has no effect and is planned to be removed from this feature set in a future release. (`MachineAPIProviderOpenStack`)
-** Insights Operator. Enables the Insights Operator, which gathers {product-title} configuration data and sends it to Red Hat. (`InsightsConfigAPI`)
+** Insights Operator. Enables the `InsightsDataGather` CRD, which allows users to configure some Insights data gathering options. It also enables the `DataGather` CRD, which allows users to run Insights data gathering as a job to create on-demand data gathering. (`InsightsConfigAPI`)
```

Comment:
Small nitpick: The Techpreview enables the `DataGather` CRD **and** the Insights data gathering as job. It isn't the CRD that enables the gathering as job. Both combined, enable the on-demand data gathering essentially.

Reply:
@JoaoFula Excellent catch. The ever-dangerous _It_. How about:

`** Insights Operator. Enables the `InsightsDataGather` CRD, which allows users to configure some Insights data gathering options. The feature set also enables the `DataGather` CRD, which allows users to run Insights data gathering on-demand. `

Reply:
That sounds good, thank you!